checkout/src/main/java/com/payoneer/checkout/util/PaymentUtils.java

```diff
@@ -401,4 +405,25 @@ public static <K, V> Map<K, V> emptyMapIfNull(Map<K, V> map) {
     public static void setTestId(View view, String type, String name) {
         view.setContentDescription(type + "." + name);
     }
+
+    /**
+     * Get the required message formatted properly
+     *
+     * @param extraElement is the element being bound on screen
+     * @return the required message for a checkbox
+     */
+    public static String getCheckboxRequiredMessage(final ExtraElement extraElement) {
+        Checkbox checkbox = extraElement.getCheckbox();
+        if (checkbox == null) {
+            return null;
+        }
+        switch (checkbox.getMode()) {
+            case CheckboxMode.REQUIRED:
+            case CheckboxMode.REQUIRED_PRESELECTED:
+                String reqMessage = checkbox.getRequiredMessage();
+                return TextUtils.isEmpty(reqMessage) ? extraElement.getName() + ".requiredMessage" : reqMessage;
```

Comment:
I try to be careful with ternary operators since it's easy to make it overly complex and hard to read. In this situation I'd do something like this:

```suggestion
                String requiredMessage = checkbox.getRequiredMessage();
                String isRequiredMessageEmpty = TextUtils.isEmpty(requiredMessage)
                String fallbackMessage = extraElement.getName() + ".requiredMessage"
                return isRequiredMessageEmpty ? fallbackMessage : requiredMessage;
```

👍, 👎, thoughts?

Reply:
Changing this now